quinn-proto/src/packet.rs

```diff
@@ -28,16 +31,16 @@ pub struct PartialDecode {
 
 #[allow(clippy::len_without_is_empty)]
 impl PartialDecode {
-    #[allow(unreachable_pub)] // fuzzing only
+    /// Decode `bytes` to create an instance of [`PartialDecode`], returning any remaining bytes
```

Comment:
This documentation is not clear about what "remaining" bytes are. How about something like this?

```rs
/// Begin decoding a QUIC packet from `bytes`, returning any trailing data not part of that packet
```

Reply:
Updated based on your suggestion.